src/services/models/ajaxModels.js

```diff
@@ -29,10 +54,14 @@ export class AjaxError {
       return 'errorCode.notAuthorized';
     } else if (this.status === 404) { // Endpoint not found
       return 'errorCode.notFound';
+    } else if (this.status === 429) { // Max Active Job Count Exceeded
```

Comment:
I don't think we can necessarily guarantee this. Might it be possible for one of our micro services to return a 429? If so we'll show the wrong error message. We don't have the right amount of granularity here to avoid those problems.

Reply:
This one isn't supposed to be there.  Sorry... I didn't undo this part.